### PR TITLE
website: added an faq page documenting cli issues

### PR DESCRIPTION
**Pull request by Kamran64**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Added an FAQ page detailing CLI issues and possible fixes.

Fixes: https://github.com/appsody/appsody/issues/86

### CODE REVIEW

**Comment by Kamran64** [comment]
Would be great to get a second opinion on the wording of some of these before merging.

File: content/docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+#FAQ
+
+Welcome to the FAQ. Here you will find a list of common questions and answers along with CLI issues you may face.
+
+##CLI Issues
+
+###1. Does Appsody support Enterprise Windows users?
+Appsody does support Enterprise Windows users, however it may require a workaround. Please follow these steps.
```
Comment (chilanti) [inline content/docs/faq.md]: I would start by saying: "Enterprise Windows users very frequently authenticate through Azure Active Directory (AAD). In those circumstances, the Docker daemon on Windows may not be authorized to access any of the folders that are owned by the AAD user. For this reason, Appsody supports Enterprise Windows users through a workaround."

File: content/docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+1. Create a new project
```
Comment (chilanti) [inline content/docs/faq.md]: and make sure that you pick a folder that is outside of your home directory, or Docker will not have access to it.

File: content/docs/faq.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+4. Change the entry ```home:``` to ```C:\my-appsody-config-dir```
+
+5. Run ```appsody run --config C:\my-appsody-config-dir\.appsody.yaml```
+
+For more information on this issue, click [here](https://github.com/appsody/appsody/issues/24).
+
+###2. Why am I getting a cross-device link error on Appsody extract?
+
+This was an issue that was faced in an older version of the software. If you are experiencing this problem, navigate to our [GitHub Page](https://github.com/appsody) and ensure you are operating on the latest version. If you would like more information on this issue, please click [here](https://github.com/appsody/appsody/issues/82).
```
Comment (chilanti) [inline content/docs/faq.md]: After "...older version of the software." I would add: "It occurred when the Appsody CLI and the Appsody project are located on different disk volumes."

File: content/docs/faq.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                                                                                                                
+###3. Why is Appsody deploy not displaying the URL of the Knative service?
+
+Although the Knative URL is not displaying, Appsody should have still been deployed successfully. This is a timing issue where ```kubectl apply``` finishes executing before all resources have been made available.
```
Comment (chilanti) [inline content/docs/faq.md]: I would introduce this issue: "Sometimes, when you run `appsody deploy`, you may see a message that fails to provide the URL of the deployed Knative service."

Comment (chilanti) [inline content/docs/faq.md]: Can you add another one for https://github.com/appsody/appsody/issues/90? This should say that Appsody 0.2.5 and older versions do not support going through an http proxy. But newer versions do - you just needs to set the HTTP_PROXY or HTTPS_PROXY env var before launching Appsody.

**Comment by Kamran64** [comment]
Thanks for the feedback @chilanti, will get right on it.

**Pull request merged**